src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -249,6 +258,62 @@ public ScheduleView AddFields(List<Revit.Schedules.SchedulableField> fields)
         }
 
         /// <summary>
+        /// Add Schedule Filters to Schedule View.
+        /// </summary>
+        /// <param name="scheduleFilters">List of Schedule Filters.</param>
+        /// <returns name="scheduleView">Schedule View.</returns>
+        public ScheduleView AddFilters(List<ScheduleFilter> scheduleFilters)
+        {
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            var currentFilters = this.InternalScheduleFilters.Select(x => new ScheduleFilter(x)).ToList();
+            foreach (var sf in scheduleFilters)
```

Comment:
I tested this and it looks that API is equally constrained as the UI version. The limit is 8 filters. If you try adding more this errors appears which is pretty self explanatory. 

![image](https://cloud.githubusercontent.com/assets/529781/23139118/e2b7a9a4-f778-11e6-81c3-1048bc0d2514.png)

Would you prefer to catch it and throw our own exception? Should we add a unit test for this condition?